benchmarks/model-based/README.md

```diff
@@ -0,0 +1,149 @@
+
+OmniSafe's navigation Benchmark evaluated the performance of OmniSafe algorithm implementations in 2 environments from the Safety-Gymnasium task suite For each algorithm and environment supported, we provide:
+
+- Default hyperparameters used for the benchmark and scripts to reproduce the results.
+- A comparison of performance or code-level details with other open-source implementations or classic papers.
+- Graphs and raw data that can be used for research purposes.
+- Log details obtained during training.
+- Some hints on how to fine-tune the algorithm for optimal results.
+
+Supported algorithms are listed below:
+
+- **[NeurIPS 2001]** [Deep Reinforcement Learning in a Handful of Trials using Probabilistic Dynamics Models (PETS))](https://arxiv.org/abs/1805.12114)
+- **[CoRL 2021]** [Learning Off-Policy with Online Planning (LOOP and SafeLOOP)](https://arxiv.org/abs/2008.10066)
+- **[AAAI 2022]** [Conservative and Adaptive Penalty for Model-Based Safe Reinforcement Learning (CAP)](https://arxiv.org/abs/2112.07701)
+- **[ICML 2022 Workshop]** [Constrained Model-based Reinforcement Learning with Robust Cross-Entropy Method (RCE)](https://arxiv.org/abs/2010.07968)
+- **[NeurIPS 2018]** [Constrained Cross-Entropy Method for Safe Reinforcement Learning (CCE)](https://proceedings.neurips.cc/paper/2018/hash/34ffeb359a192eb8174b6854643cc046-Abstract.html)
+
+## Safety-Gymnasium
+
+We highly recommend using ``safety-gymnasium`` to run the following experiments. To install, in a linux machine, type:
+
+```bash
+pip install safety_gymnasium
+```
+
+## Run the Benchmark
+
+You can set the main function of ``examples/benchmarks/experimrnt_grid.py`` as:
+
+```python
+if __name__ == '__main__':
+    eg = ExperimentGrid(exp_name='Model-Based-Benchmarks')
+
+    # set up the algorithms.
+    model_based_base_policy = ['LOOP', 'PETS']
+    model_based_safe_policy = ['SafeLOOP', 'CCEPETS', 'CAPPETS', 'RCEPETS']
+    eg.add('algo', model_based_base_policy + model_based_safe_policy)
+
+    # you can use wandb to monitor the experiment.
+    eg.add('logger_cfgs:use_wandb', [False])
+    # you can use tensorboard to monitor the experiment.
+    eg.add('logger_cfgs:use_tensorboard', [True])
+    eg.add('train_cfgs:total_steps', [1000000])
+
+    # set up the environment.
+    eg.add('env_id', [
+        'SafetyPointGoal1-v0-modelbased',
+        'SafetyCarGoal1-v0-modelbased',
+        ])
+    eg.add('seed', [0, 5, 10, 15, 20])
+
+    # total experiment num must can be divided by num_pool
+    # meanwhile, users should decide this value according to their machine
+    eg.run(train, num_pool=5)
+```
+
+After that, you can run the following command to run the benchmark:
+
+```bash
+cd examples/benchmarks
+python run_experiment_grid.py
+```
+
+You can set the path of ``examples/benchmarks/experimrnt_grid.py`` :
+example:
+
+```python
+path ='/home/username/omnisafe/omnisafe/examples/benchmarks/exp-x/Model-Based-Benchmarks'
```

Comment:
May be relative path better. e.g. `./examples/benchmarks/exp-x/Model-Based-Benchmarks`